### PR TITLE
[stable/elasticsearch] Split master and data persistent storage class

### PR DESCRIPTION
**Pull request by mikelorant**
This allows for different provisioner and provisioner type for the master and data persistent storage. This is especially useful when trying to use AWS EBS `st1` which requires volumes greater than 500GB however master nodes can still be `gp2` and be small volumes of 4GB.

### CODE REVIEW

File: stable/elasticsearch/Chart.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ appVersion: "1.0"
 description: A Helm chart for Elasticsearch
 name: elasticsearch
 icon: https://www.elastic.co/static/images/elastic-logo-200.png
-version: 1.0.0-elasticsearch6.5.1
+version: 1.1.0-elasticsearch6.5.1
 maintainers:
   - name: FFX Blue Operations
     email: blueops@fairfaxmedia.com.au
```
Comment (egeland) [inline stable/elasticsearch/Chart.yaml]: Could you please use the github@fairfaxmedia.com.au account - see the k8s-secret-updater chart

**Review by egeland — CHANGES_REQUESTED** [review]

File: stable/elasticsearch/Chart.yaml (hunk 1 above)
Comment (mikelorant) [reply stable/elasticsearch/Chart.yaml]: Will do.

Comment (egeland) [reply stable/elasticsearch/Chart.yaml]: https://github.com/fairfaxmedia/charts/blob/bf5058f62c164f3b4ee19c783230c0b0ff196642/stable/k8s-secret-updater/Chart.yaml#L6-L7

Comment (mikelorant) [reply stable/elasticsearch/Chart.yaml]: Change made.

**Review by egeland — APPROVED** [review]

**Pull request merged**